program/tests/duplicate_block_proof.rs

```diff
@@ -254,11 +278,6 @@ async fn valid_proof_coding() {
     let shred2 =
         new_rand_coding_shreds(&mut rng, next_shred_index, 10, &shredder, &leader)[1].clone();
 
-    assert!(
-        ErasureMeta::check_erasure_consistency(&shred1, &shred2),
-        "Expected erasure consistency failure",
-    );
```

Comment:
Did you mean to remove this check?

Reply:
Yeah it's not completely accurate to say it's an erasure consistency failure here, the real failure is the merkle root of shred1 and shred2 will not be consistent.